ApeWorX/ape
# fix: require local network when starting ephemeral geth

### What I did

Require local network when starting an ephemeral local geth

### How I did it

Noticed we always start the ephemeral geth process, regardless of what network we are on.
This PR will limit it to just local

### How to verify it

Go to a project with tests or something and run `ape test --network ::geth` and make sure that still works

### Checklist
